app/controllers/admin/api_docs/account_api_docs_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Admin::ApiDocs::AccountApiDocsController < Admin::ApiDocs::BaseController
+  class NotImplementedServiceScopeError < NotImplementedError; end
```

Comment:
Code Climate complain for reason

https://ruby-doc.org/core-2.3.7/NotImplementedError.html